Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -4702,13 +4723,16 @@
 					"\"${PODS_ROOT}/leveldb-library/include\"",
 				);
 				INFOPLIST_FILE = "App/iOS/Firestore-Info.plist";
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
```

Comment:
Sorry to nitpick in the project files, but did you add this line? If so, why?

Reply:
Not sure why it is there. Removed it.